ftw/simplelayout/tests/test_contentpage_behavior.py

```diff
@@ -41,3 +42,13 @@ def test_title_shows_if_show_title_is_true(self, browser):
         self.assertEqual(
             ['A page'],
             browser.css('.documentFirstHeading').text)
+
+    @browsing
+    def test_show_hide_title_field_is_only_for_admins(self, browser):
```

Comment:
`only_for_admins` may be misleading since this is a side-effect here:
In this test it is "only for admins" because we have no active workflow and therefore the `rolemap.xml` counts, which does not grant contributors / editors.

But in production we will have a workflow, usually with `ftw.lawgiver`. In the `lawgiver.zcml` you are mapping the permission to the `edit` action group, therefore the standard behavior will be that all editors can hide the title.

Therefore I think the test name may be misleading.
How can we address that?
Should we just add a docstring to the test describing this issue?

Or is the `rolemap.xml` wrong because it does not go along with the `edit` action group? Then we should probably add editors / contributors in the `rolemap.xml` and then reconfigure the permission here so that we are explicitly building the situation in the test.

@maethu what do you think?